nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/SNMPUtils.java

```diff
@@ -191,26 +164,6 @@ private static void addAttributeFromVariable(VariableBinding variableBinding, Ma
     }
 
     /**
-     * Will validate if provided name corresponds to valid SNMP property.
-     * @param name the name of the property
-     * @return 'true' if valid otherwise 'false'
-     */
-    public static boolean isValidSnmpPropertyName(String name) {
```

Comment:
I know this method isn't used in your improved attribute populator, but it shouldn't be removed as it is a public method in a util class.